content/en/blog/_posts/2023-08-04-kubernetes-1.28-blog.md

```diff
@@ -0,0 +1,306 @@
+    
+Many common machine learning frameworks, such as Tensorflow and JAX, require unique pods per index.
+With the older behaviour, if a pod that belongs to an `Indexed` Job enters a terminating state (due to preemption, eviction or other external factors), a replacement pod is created but then immediately fails to start due
+to the clash with the old pod that has not yet shut down.
+
+Having a replacement Pod appear before the previous one fully terminates can also cause problems
+in clusters with scarce resources or with tight budgets. These resources can be difficult to obtain so 
+can take a long time to find resources and they may only be able to find nodes once the existing pods
+have been terminated. If cluster autoscaler is enabled, early creation of replacement Pods might produce undesired scale-ups.
+
+To learn more, read [Delayed creation of replacement pods](/docs/concepts/workloads/controllers/job/#delayed-creation-of-replacement-pods)
+in the Job documentation.
+  
+## Job retry backoff limit, per index
+    
+This extends the Job API to support indexed jobs where the backoff limit is per index, and the Job can continue execution despite some of its indexes failing.
+    
+Currently, the indexes of an indexed job share a single backoff limit. When the job reaches this shared backoff limit, the job controller marks the entire job as failed, and the resources are cleaned up, including indexes that have yet to run to completion.
+
+As a result, the existing implementation did not cover the situation where the workload is truly
+[embarrassingly parallel](https://en.wikipedia.org/wiki/Embarrassingly_parallel): each index is
+fully independent of other indexes.
+
+For instance, if indexed jobs were used as the basis for a suite of long-running integration tests, then each test run would only be able to find a single test failure.
+
+    
+For more information, read [Handling Pod and container failures](/docs/concepts/workloads/controllers/job/#handling-pod-and-container-failures) in the Kubernetes documentation.
+   
+## cAdvisor-less, CRI-full Container And Pod Stats
+
+This encompasses two related pieces of work (changes to the kubelet's `/metrics/cadvisor` endpoint, and improvements to the replacement _summary_ API).
+
+There are two main APIs that consumers use to gather stats about running containers and pods: summary API and /metrics/cadvisor. The Kubelet is responsible for implementing the summary API, and cadvisor is responsible for fulfilling /metrics/cadvisor.
+
+This aims to enhance CRI implementations to be able to fulfill all the stats needs of Kubernetes. At a high level, there are two pieces of this:
+
+- Enhance the CRI API with enough metrics to be able to supplement the pod and container fields in the summary API directly from CRI.
+    
+- Enhance the CRI implementations to broadcast the required metrics to fulfill the pod and container fields in the /metrics/cadvisor endpoint.
```

Comment:
Rewrite this as if the feature is already released (it will have been). Also, use Markdown (eg backticks around URI paths).